source/crud/update.txt

```diff
@@ -52,38 +51,29 @@ The pattern expects you to:
 * Specify the field and value changes.
 * Specify options, if you must modify the method behavior.
 
-The driver provides the following methods to modify documents:
+The driver provides the following methods to update documents:
 
-* ``UpdateByID()``
-* ``UpdateOne()``
-* ``UpdateMany()``
-* ``ReplaceOne()``
-* ``BulkWrite()`` *(not discussed in this guide)*
-* ``FindOneAndUpdate()`` *(not discussed in this guide)*
-* ``FindOneAndReplace()`` *(not discussed in this guide)*
+* ``UpdateByID()``: Updates a single document based on its ``_id``.
+* ``UpdateOne()``: Updates a single document.
+* ``UpdateMany()``: Updates multiple documents.
 
-A Note About ``_id``
-~~~~~~~~~~~~~~~~~~~~
+A Note About _id
+~~~~~~~~~~~~~~~~~
 
 Each document in a MongoDB collection has a unique and immutable ``_id``
-field. You cannot use update and replace operations to change the
-``_id`` field. If you attempt to change this field, the update and
-replace methods return a ``WriteError``.
-
-.. _golang-update-documents:
+field. You cannot use update operations to change the
+``_id`` field. If you attempt to change this field, the update
+methods return a ``WriteError``.
 
-Update
-------
+Sample Data
+~~~~~~~~~~~
```

Comment:
S; It looks like there's no content under this heading, is it missing or is this heading in the wrong place?